readme.txt

```diff
@@ -39,6 +39,9 @@ When opening a bug on GitHub, please give us as many details as possible.
 
 == Changelog ==
 
+= 2021.03.31 - version 2.3.5 =
+ * Fix - critical issue for pre 5.0.0 WC sites

```

Comment:
Would be good to have a bit more detail in changelog entry, e.g. `Fix - Avoid critical error with new WooCommerce Nav in Woo < 5.0`.

Since the release has already gone out - no biggie 😄